DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/SubstituteProcessor.cs

```diff
@@ -0,0 +1,73 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.Anonymizer.Core.Exceptions;
+using Microsoft.Health.Dicom.Anonymizer.Core.Model;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors
+{
+    public class SubstituteProcessor : IAnonymizerProcessor
+    {
+        private readonly string _replaceWith = "Anonymous";
+
+        public SubstituteProcessor(JObject settingObject)
+        {
+            EnsureArg.IsNotNull(settingObject, nameof(settingObject));
+
+            if (settingObject.TryGetValue("replaceWith", StringComparison.OrdinalIgnoreCase, out JToken replaced))
```

Comment:
Looks like we can have a SubstituteSetting here instead of reading json directly? Just in case we have more parameters in the future.